spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.feature 'Create Comment' do
+
+  context 'when a visitor tries to create a Comment' do
+
+    before do
+      @pic = FactoryGirl.create(:picture)
+      visit '/'
+    end
+
+    scenario "he can't do it" do
+      click_link 'Show'
+      fill_in :comment_body, with: 'Luv it!'
+      click_button 'Create Comment'
+      expect(page).to have_content('You need to sign in or sign up before continuing.')
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)